### ISSUE TITLE
Right Click Functions Not Working

### ISSUE THREAD

**Issue by Emilis23** [ldtteam/minecolonies#5551]
### Prerequisites 

- [x] I am running the latest **alpha** version of MineColonies and Structurize for my Minecraft version.
- [x] I checked the MineColonies/Structurize [wiki](https://wiki.minecolonies.ldtteam.com/) and made sure my issue is not covered there.
- [x] I made sure that **this issue is not a duplicate of any existing issue.**

### Context 

- Minecraft Version: 1.16.1
- MineColonies Version: 0.13.123-ALPHA
- Structurize Version: 0.13.44-ALPHA
- Related Mods and their Versions: Had issue with about 77 other mods, but then removed all but OPTIFINE 1.16.1 HD U G2 and getting same issues

## Expected behavior

Right clicking citizens opens a hud where I can access their inventories and check other details like requests.

## Steps to reproduce the issue

Initially I ran into this after starting a colony on a heavily modded game with edits in the config files for both Minecolonies and other mods. I figured it was a problem with a conflict or a bad alpha release. I've reproduced the problem now on several alpha versions of minecolonies (and the proper updated versions of structurize) sense then though and it persists. I also tried starting a new game after uninstalling every other mod besides Optifine and replacing the config files with their originals, but it still persists. 

1. Install most recent version of minecolonies and structurize
2. start new world
3. set up supply camp, set up town hall, wait for first citizen to arrive.
4. try right clicking citizen.
5. get frustrated at lack of hud.

I figure I've done something dumb here given the amount of changes I like to make in my game and don't realize it, but I've tried trouble shooting the problem to the point that I'm at a loss.

### Logs

no crashlogs to deal with.

### Notes

---
## Viewers

* Add a thumbs-up to the bug report if you are also affected. This helps the bug report become more visible to the team and doesn't clutter the comments.
* Add a comment if you have any insights or background information that isn't already part of the conversation.

**Comment by ravenbuilder934** [comment]
Remove OptiFine, it's their fault

**Comment by Emilis23** [comment]
Yep, that did the trick. Thanks for the quick fix! Figures it'd be the one mod I left installed. I just didn't want to mess with something where I'd have to remove my shaders and resource packs, and I figured that'd be unrelated enough to not be causing a problem.

Glad to know where the issue lies. Now I just hope Optifine can fix it. Don't really want to give up the graphics, but I'd rather have the game play if I've got to choose.
